package.json

```diff
@@ -23,8 +23,9 @@
     "install:sass": "npm run install:sass:webcomponent && npm run install:sass:demo",
     "install:sass:webcomponent": "sass ../../src/style.scss ../../src/style.css --load-path ../../node_modules/@govflanders",
     "install:sass:demo": "sass demo.scss ../../demo/demo.css --load-path ../../node_modules/@govflanders",
-    "install:copy": "npm run install:copy:install && npm run install:copy:test && npm run install:copy:release && npm run install:copy:demo:js && npm run install:copy:demo:css && npm run install:copy:prismjs && npm run install:copy:codeowner && npm run install:copy:github && npm run install:copy:wct && npm run install:copy:mocharc",
+    "install:copy": "npm run install:copy:install && npm run install:copy:test && npm run install:copy:release && npm run install:copy:demo:js && npm run install:copy:demo:css && npm run install:copy:prismjs && npm run install:copy:codeowner && npm run install:copy:github && npm run install:copy:wct && npm run install:copy:mocharc && npm run install:copy:eslint",
     "install:copy:install": "npm i copyfiles@2.1.0 --save",
+    "install:copy:eslint": "cp .eslintrc.json ../../",
```

Comment:
Nu kopiëren we die enkel, zodat de editors er mee aan de slag kunnen, maar gaan we dit ook ergens controleren bij een build/release/install, of is dit een ander ticket?

Reply:
Dat is de bedoeling ja, maar die run scripts gaan in elke component apart moeten gezet worden.

Reply:
Kunnen we zo niet eens globaal doen zoals: `eslint ../../src/**/*.js` bij elke install?